installers/agent/release/agent.sh

```diff
@@ -101,15 +98,18 @@ AGENT_STARTUP_ARGS="-Dcruise.console.publish.interval=10 -Xms$AGENT_MEM -Xmx$AGE
 if [ "$TMPDIR" != "" ]; then
     AGENT_STARTUP_ARGS="$AGENT_STARTUP_ARGS -Djava.io.tmpdir=$TMPDIR"
 fi
+if [ "$USE_URANDOM" != "false" ] && [ -e "/dev/urandom" ]; then
+    AGENT_STARTUP_ARGS="$AGENT_STARTUP_ARGS -Djava.security.egd=file:/dev/./urandom"
```

Comment:
Due to this [old bug](http://bugs.java.com/view_bug.do?bug_id=6202721), but that has been fixed in later versions. I guess it can be changed now.